src/templates/drizzle/labelheader.hbs

```diff
@@ -0,0 +1,20 @@
+{{#block "heading"}}
+  <h4 class="
+   drizzle-u-inlineBlock
+   drizzle-u-alignMiddle
+   drizzle-u-textNoWrap
+   drizzle-u-padRight">
+    <a href="./{{key}}.html">{{title}}</a>
+  </h4>
+{{/block}}
+{{#if labels}}
```

Comment:
@erikjung: I added this logic into this partial. What are your thoughts? Not all label headings will have labels.